pkg/network/java/hotspot.go

```diff
@@ -304,6 +304,15 @@ func (h *Hotspot) Attach(agentPath string, args string, uid int, gid int) error
 	}
 	defer cleanConn()
 
+	i := 0
+	var args string
+	for k, v := range argsKV {
+		args += k + "=" + v
+		if i != len(argsKV)-1 {
+			args += " "
+		}
+		i++
+	}
```

Comment:
why making the map in the first place if we transform into string?
let's simplify and get all the arguments as 1 string